CHANGELOG-pre1.0.0.md

```diff
@@ -0,0 +1,1363 @@
+# CHANGELOG of versions before 1.0.0
```

Comment:
Does this file need to be added as a new file?

Reply:
Deleted, thank you.